.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on: push
+
+jobs:
+  molecule:
+    runs-on: ubuntu-18.04
```

Comment:
Just curious why ubuntu-18.04 vs -latest or -20.04? I assume some sort of compatibility thing? Maybe you're just trying to 1:1 from whatever Travis-CI was using.

Reply:
At the moment, this is equivalent to ubuntu-latest on GHA, and at some point in the future, it'll use ubuntu-20.04